api/auth.py

```diff
@@ -12,7 +12,7 @@
     BearerTransport,
     JWTStrategy,
 )
-from .config import AuthSettings
+from .models import AuthSettings
```

Comment:
Why the settings classes have been moved to `models`?

Reply:
These are `Settings` classes and not usual object models. That's why we have it in the `.config` file. These are specific to API only. That's why we should keep it in the API.